GUI/sanity-test-halo/jest-puppeteer/jest-puppeteer.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    launch: {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Seagate/cortx-experiments/pullRequest?prid=9333958)
```suggestion
  launch: {
```